Sources/Commands/Workspace.swift

```diff
@@ -156,6 +170,9 @@ public class Workspace {
     /// The path for working repository clones (checkouts).
     let checkoutsPath: AbsolutePath
 
+    /// The path where packages which are put in edit mode are checked out.
+    let editsPath: AbsolutePath
```

Comment:
`editablesPath` is more grammatically correct. (`editsPath` would store individual edits)